### ISSUE TITLE
"Italian" workshopper

### ISSUE THREAD

**Issue by marcopiraccini** [nodeschool/italy#17]
Guys, it's time to become more active in the nodeschool community!
Any idea for a new workshopper? (I wrote "italian", but of course, once started the project anyone in the universe will help us :)). 

Note that there is alredy a catalog of "requested" workshoppers: https://github.com/nodeschool/workshoppers/issues?q=label%3Astatus%3Arequested

**Comment by simoneb** [comment]
Working on .NET for some of my projects I started working on one about Edge.js, but it doesn't have an underlying Italian theme :smile:

**Comment by mcollina** [comment]
@marcopiraccini a MQTT.js workshopper? :)

**Comment by marcopiraccini** [comment]
@mcollina why not

**Comment by simoneb** [comment]
@mcollina indeed

**Comment by mcollina** [comment]
@marcopiraccini @simoneb set it up! :) I can create a repo in the http://github.com/mqttjs organization if you want and you there too!

**Comment by simoneb** [comment]
@mcollina go ahead

**Comment by mcollina** [comment]
Ideas for the name? Something fancy like makemehapi!

**Comment by simoneb** [comment]
let's see, the name mqtt doesn't lend itself too well but (with or w/o the dashes):
- pub-sub-me
- queue-me-up

or a mix of the buzzwords :smile:

**Comment by keul** [comment]
Not sure if it's a little off-topic for a workshop but what about **io.js**? Why it exists? What it's doing better compared to node.js?

Another issue: node 0.12/ES6

**Comment by mcollina** [comment]
How about: mqttify-yourself? wow-mqtt?

**Comment by mcollina** [comment]
@keul given io.js will get into node.js around September as node 4, probably it's a bad idea.

**Comment by simoneb** [comment]
@mcollina wow-mqtt sounds good, not easy to spell out but cool nonetheless :smile:

**Comment by marcopiraccini** [comment]
+1 for wow-mqtt

**Comment by gpad** [comment]
+1 for wow-mqtt

2015-07-15 11:48 GMT+02:00 Marco notifications@github.com:

**Comment by mcollina** [comment]
Done! I'm pulling in @nvcexploder, we were talking about doing a MQTT.js workshopper at NodeSummit.

**Comment by diegobent** [comment]
hello guys! mqtt is great!
+1 wow-mqtt

Now i'm working with loopback framework (rest api based on Express) and it's very cool but perhaps it would be too complex for a workshopper... someone else has tried it?

**Comment by lambrojos** [comment]
@diegobent i have been working with loopback for ~6 months and I think i have worked around most of its documentarion flaws.. so count me in :)

**Comment by mcollina** [comment]
Ping @bajtos, he probably have some recommendations to run an event on Loopback!

**Comment by lambrojos** [comment]
Yes I know for sure he has a nice sample application about whiskey.. could be helpful!

**Comment by bajtos** [comment]
I have a LoopBack + AngularJS workshop, see https://github.com/bajtos/loopback-workshop

**Comment by mcollina** [comment]
@bajtos that looks nice, thanks for linking!

**Comment by kildem** [comment]
+1 wow-mqtt

**Comment by claudiopro** [comment]
Ciao a tutti! Workshopper adesso parla italiano :grin: https://github.com/workshopper/workshopper-adventure/pull/8

**Comment by claudiopro** [comment]
By the way is there anyone in the Palermo/Sicily area? I'd like to start a NodeSchool chapter here.
